archive/archive.go

```diff
@@ -628,7 +628,7 @@ func (archive *Archive) AddRecordedPublication(channel string, stream int32) (*a
 
 	archive.mtx.Lock()
 	defer archive.mtx.Unlock()
-	if err := archive.Proxy.StartRecordingRequest(correlationID, stream, true, true, sessionChannel); err != nil {
+	if err := archive.Proxy.StartRecordingRequest(correlationID, stream, true, false, sessionChannel); err != nil {
```

Comment:
btw, did we check that this `true/false` thing is accurate?

Reply:
Let's not include this change into this PR, I'll split it to make the commit log cleaner